xinference/model/llm/utils.py

```diff
@@ -446,6 +446,7 @@ def _to_chat_completion_chunk(cls, chunk: CompletionChunk) -> ChatCompletionChun
                     "index": i,
                     "delta": {
                         "content": choice["text"],
+                        "tool_calls": choice.get("tool_calls"),
```

Comment:
When there is no `tool_call` field, should we just not add this field the the chat completion, rather than a `None` here? (in order to be more compatible with OpenAI API)

Reply:
We can replace it with `**({"tool_calls": choice["tool_calls"]} if "tool_calls" in choice else {})`